README.md

```diff
@@ -1,7 +1,17 @@
-# opentelemetry-go-contrib
+# OpenTelemetry-Go Contrib
 
-This repo contains packages that facilitate instrumenting commonly
-used libraries with OpenTelemetry for distributed tracing and
-monitoring.
+[![Circle CI](https://circleci.com/gh/open-telemetry/opentelemetry-go-contrib.svg?style=svg)](https://circleci.com/gh/open-telemetry/opentelemetry-go-contrib)
+[![Docs](https://godoc.org/go.opentelemetry.io/contrib?status.svg)](https://pkg.go.dev/go.opentelemetry.io/contrib)
+[![Go Report Card](https://goreportcard.com/badge/go.opentelemetry.io/contrib)](https://goreportcard.com/report/go.opentelemetry.io/contrib)
+[![Gitter](https://badges.gitter.im/open-telemetry/opentelemetry-go-contrib.svg)](https://gitter.im/open-telemetry/opentelemetry-go-contrib?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
```

Comment:
This gitter link goes to a 404 page for me.

Reply:
Ah, good catch. :+1: